locale/en_US/manager.po

```diff
@@ -2141,3 +2141,27 @@ msgstr "The full names of the authors"
 
 msgid "emailTemplate.variable.submission.submissionUrl"
 msgstr "The URL to the submission in the editorial backend"
+
+msgid "mailable.mailDiscussionMessage.name"
+msgstr "Discussions"
+
+msgid "mailable.mailDiscussionMessage.description"
+msgstr "Automatic email sent to participants of a new or updated discussion"
```

Comment:
This email is automatically sent to discussion participants when a new message is added to the discussion.